src/battleground/ImageName.js

```diff
@@ -1,5 +1,10 @@
 //@flow strict
 
-type ImageName = 'DIRT_BACKGROUND' | 'BLUE_CHASSIS_1' | 'RED_GUN_1' | 'GRAY_TREAD_1';
+type ImageName = 
+	'DIRT_BACKGROUND' | 
+	'WALL' | 
+	'BLUE_CHASSIS_1' | 
```

Comment:
Not a huge issue right now, but soon we will need the components to be named correctly. I assume you are probable waiting on the ability to pull tanks first.